### PR TITLE
updated app registration steps

### PR DESCRIPTION
**Pull request by kjyam98**
@Jackson-Woods updated the app registration steps...please review PR and merge. ADO task 2012

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,21 +33,25 @@ To use this sample, you need the following:
...
 
-3. Enter a name for the app, and choose **Create application**. 
+3. In the **Name** section, enter a meaningful application name that will be displayed to users of the app
+
+1. In the **Supported account types** section, select **Accounts in any organizational directory and personal Microsoft accounts (e.g. Skype, Xbox, Outlook.com)**  
+
+1. Select **Register** to create the application. 
 	
-   The registration page displays, listing the properties of your app.
+   The application's Overview page the properties of your app.
```
Comment (Jackson-Woods) [inline README.md]: @kjyam98 Think this line is missing a verb - shows?

File: README.md [hunk 2]
```diff
@@ -33,21 +33,25 @@ To use this sample, you need the following:
...
+4. Copy the *Application (client) Id*. This is the unique identifier for your app. 
 
-5. Under **Platforms**, choose **Add Platform**.
+5. In the list of pages for the app, select  **Authentication**.
 
-6. Choose **Web**.
+6. In the *Redirect URIs* section, choose **Web** from the *Type* dropdown and enter *http://localhost:3000/* as the *Redirect URI*. 
 
-7. Make sure the **Allow Implicit Flow** check box is selected, and enter *http://localhost:3000/* as the Redirect URI. 
+1. Under *Advanced Settings* enable implicit flow by checking the *Access tokens* and *ID tokens* boxes
```
Comment (Jackson-Woods) [inline README.md]: Can you please phrase as "implicit grant flow" to match portal verbiage?

File: README.md [hunk 3]
```diff
@@ -33,21 +33,25 @@ To use this sample, you need the following:
...
+   The application's Overview page the properties of your app.
 
-4. Copy the Application Id. This is the unique identifier for your app. 
+4. Copy the *Application (client) Id*. This is the unique identifier for your app. 
 
-5. Under **Platforms**, choose **Add Platform**.
+5. In the list of pages for the app, select  **Authentication**.
 
-6. Choose **Web**.
+6. In the *Redirect URIs* section, choose **Web** from the *Type* dropdown and enter *http://localhost:3000/* as the *Redirect URI*.
```
Comment (Lauragra) [inline README.md]: Please apply bold style rather than italics to UI elements.

File: README.md (hunk 1 above)
Comment (kjyam98) [reply README.md]: Updated

File: README.md (hunk 2 above)
Comment (kjyam98) [reply README.md]: Updated

File: README.md (hunk 3 above)
Comment (kjyam98) [reply README.md]: Updated

**Pull request merged**